curriculum/challenges/english/07-scientific-computing-with-python/learn-lambda-functions-by-building-an-expense-tracker/65822bd82d708c4895080c35.md

```diff
@@ -14,7 +14,14 @@ my_dict = {'amount': 50.0, 'category': 'Food'}
 my_dict['amount'] # 50.0
 ```
 
-You are currently interpolating the `expense` dictionary in your f-string. Modify the f-string expression to access the value of the `'amount'` key and the `'category'` key in the `expense` dictionary. Remember to use double quotes within your single-quoted f-string and vice versa.
+You are currently interpolating the `expense` dictionary in your f-string. Modify the f-string expression to access the value of the `'amount'` key and the `'category'` key in the `expense` dictionary.
+
+An important thing to know is that the same type of quote used to define a string cannot be used inside it. For example, the string `'I'm a string!'` is not valid. To use the single quote inside that string you should either:
+
+- Escape the quote by prepending a backlash to it: `'I\'m a string!'`
+- Or use double quotes to define the string: `"I'm a string!"`
+
+with the latter being the preferred option.
```

Comment:
Reordered the paragraphs so that the instruction for the step comes at the end.

```suggestion
In Python, an important thing to know is that the same type of quote used to define a string cannot be used inside it. For example, the string `'I'm a string!'` is not valid. To use the single quote inside that string you should either:

- Escape the quote by prepending a backlash to it: `'I\'m a string!'`.
- Or use double quotes to define the string: `"I'm a string!"`.(Preferred)

You are currently interpolating the expense dictionary in your f-string. Modify the f-string expression to access the value of the `'amount'` key and the `'category'` key in the expense dictionary.
```